public/types.ts

```diff
@@ -67,9 +67,9 @@ export interface State {
 
   backgroundColor: [number, number, number];
 
-  flipX: number;
-  flipY: number;
-  flipZ: number;
+  flipX: -1 | 1;
+  flipY: -1 | 1;
+  flipZ: -1 | 1;
```

Comment:
this is increasingly out of scope but now these suddenly look like booleans to me 🤣

Reply:
You're killing me here Dan 😭

Reply:
I can make this change though, I just need to check that it doesn't touch external interfaces

Reply:
sorry, it's not a good candidate to happen in this PR, if at all.   no real reason to change it